### PR TITLE
app-backup/mkstage4: new ebuild

### PR DESCRIPTION
**Pull request by TheChymera**
Package-Manager: Portage-2.3.31, Repoman-2.3.9

### CODE REVIEW

File: app-backup/mkstage4/metadata.xml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (Zlogene) [inline app-backup/mkstage4/metadata.xml]: Add proxy-mainteiners last

File: app-backup/mkstage4/mkstage4-0.1.ebuild [hunk 2]
```diff
@@ -0,0 +1,22 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="6"
```
Comment (Zlogene) [inline app-backup/mkstage4/mkstage4-0.1.ebuild]: no quotes here please

File: app-backup/mkstage4/mkstage4-0.1.ebuild [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+SLOT="0"
+KEYWORDS="~amd64"
+IUSE=""
+
+DEPEND=""
+RDEPEND="app-shells/bash
+	app-arch/tar"
+
+src_install() {
+	exeinto /usr/bin/
```
Comment (Zlogene) [inline app-backup/mkstage4/mkstage4-0.1.ebuild]: why? dobin installs into `/usr/bin`, is there the reason why you did not run into this?

File: app-backup/mkstage4/mkstage4-0.1.ebuild [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+	newexe "${S}"/mkstage4.sh mkstage4
```
Comment (Zlogene) [inline app-backup/mkstage4/mkstage4-0.1.ebuild]: why?

File: app-backup/mkstage4/mkstage4-9999.ebuild [hunk 5]
```diff
@@ -0,0 +1,24 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="6"
```
Comment (Zlogene) [inline app-backup/mkstage4/mkstage4-9999.ebuild]: same as versioned ebuild

**Comment by mgorny** [comment]
@TheChymera, ping. Please answer @Zlogene's review.

File: app-backup/mkstage4/metadata.xml (hunk 1 above)
Comment (TheChymera) [reply app-backup/mkstage4/metadata.xml]: added

File: app-backup/mkstage4/mkstage4-9999.ebuild [hunk 6]
```diff
@@ -0,0 +1,24 @@
...
+SLOT="0"
+KEYWORDS=""
+IUSE=""
+
+DEPEND=""
+RDEPEND="app-shells/bash
+	app-arch/tar"
+
+src_install() {
+	exeinto /usr/bin/
```
Comment (Zlogene) [inline app-backup/mkstage4/mkstage4-9999.ebuild]: still wrong, see earlier review

Comment (TheChymera) [reply app-backup/mkstage4/mkstage4-9999.ebuild]: sorry, missed that.

**Pull request closed without merging**